lizmap/www/assets/js/map.js

```diff
@@ -168,6 +168,51 @@ var lizMap = function() {
    */
   var editionPending = false;
 
+  /**
+   * Get the metadata written in the configuration file by the desktop Lizmap plugin.
+   *
+   * This method should be used EVERY TIME we need to add "if" conditions
+   * to adapt the code for configuration parameters changes across versions.
+   * This will ease in the future the review of the code to remove all the "if"
+   * conditions: we will just need to search for "getLizmapDesktopPluginMetadata"
+   * and not: "if 'someproperty' in someconfig".
+   *
+   * For very old configuration files, which have not the needed metadata, we
+   * return fake versions for each property.
+   *
+   * Dependencies:
+   * config
+   */
+  function getLizmapDesktopPluginMetadata()
+  {
+    // Default fake versions if the properties does not yet exist in configuration file
+    var plugin_metadata = {
+      lizmap_plugin_version: 30108,
```

Comment:
Note for myself, I need to add a new key lizmap plugin version as int ...
In the CFG, it's like : 
```json
    "metadata": {
        "qgis_desktop_version": 31615,
        "lizmap_plugin_version": "3.7.7",
        "lizmap_web_client_target_version": 30500,
        "project_valid": true
    },
```